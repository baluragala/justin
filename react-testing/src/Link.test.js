import React from "react";
import Enzyme, { shallow, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { shallowToJson } from "enzyme-to-json";
import Link from "./Link";

Enzyme.configure({ adapter: new Adapter() });

describe("Link", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<Link title="mockTitle" url="mockurl" />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
