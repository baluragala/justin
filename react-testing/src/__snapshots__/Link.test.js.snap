// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render correctly 1`] = `
<a
  href="mockurl"
  onClick={[Function]}
>
  mockTitle
</a>
`;
